util/testutil/integration/containerd.go

```diff
@@ -64,7 +64,7 @@ func (c *containerd) New() (sb Sandbox, cl func() error, err error) {
 		return nil, nil, err
 	}
 
-	builddSock, stop, err := runBuildd([]string{"buildd-containerd", "--containerd-worker-addr", address}, logs)
+	builddSock, stop, err := runBuildd([]string{"buildd.containerd_worker_only", "--containerd-worker-addr", address}, logs)
```

Comment:
This can be follow-up: Test with regular `buildd` (with different flags) as this is the recommended way to use `buildd`